### PR TITLE
Some bugfixes

### LINKED ISSUE #11960: Using thermite creates an abyss into the nether realms
**Problem Description**:
Splashing thermite on walls makes them turn into a black void that cannot be interacted with

**What did you expect to happen**: 
To be able to put thermite on a wall and melt it

**What happened instead**:
The wall became an abyss that I couldn't interact with

**Why is this bad/What are the consequences:**
Severely limits the use of thermite because walls now become inaccessible when doused 

**Steps to reproduce the problem**:
Take thermite, splash it on a wall, watch as it becomes a black box that you can't even alt click on

**When did the problem start happening**:
Not sure, only just encountered it today

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?)**:
When other nearby walls were removed it seemed to restore a bit of the wall, perhaps this happened due to the way wall textures join together
Screenshot (25)

### LINKED ISSUE #12997: Thermite can no longer be triggered by a welder.
**Issue Description**:

When trying to use thermite on a reinforced wall it no longer is triggerable by a welder.

**What did you expect to happen**: 

The current patch of the game has made this a problem, I'm not sure as I don't know anything about coding why the issue is going on. Perhaps it's literally just the way the game thinks you are trying to use the tool on the wall and cancels the action?
 I genuinely don't know, I'm guessing.
**What happened instead**:

It just doesn't work when it used to.

**Why is this bad/What are the consequences**:

Thermite is important to remove walls quickly.

**Steps to reproduce the problem**:

Use thermite on an reinforced wall then try to heat it using a welder.

**When did the problem start happening**:

This release. About three days ago.

**Extra information**:

Nothing more to say.

### LINKED ISSUE #13023: Disposal Chutes turns into Delivery chutes
**Issue Description**:
Upon removing a disposal chute from the floor it turns into a delivery chute.

**What did you expect to happen**: 
I expected it to be removed from the floor to be able to be moved and placed into a new location.

**What happened instead**:
Upon removing it from the floor it turned into a delivery chute.

**Why is this bad/What are the consequences**:
Disposal Chutes cannot be moved and when they get taken off the floor they turn into a different entity 

**Steps to reproduce the problem**:
Weld, Unwrench Pipe from underneath disposal chute, Unscrew power connectors from disposal chute, unweld disposal chute from floor. Disposal Chute is now a delivery chute.

**When did the problem start happening**:
This is a very recent bug as it wasn't occurring a month ago I believe, I found out about this on 2/23/2020 at around 10:20 PM PST

**Extra information**:
It's repeatable and occur with any disposal chute as I tested it on the experimenter one as well to see if it's a reoccurring bug.

### LINKED ISSUE #13035: Disposals Delivery Chute can't be dissasembled
**Issue Description**:
Usually to dissasemble a delivery chute you use a screwdriver and then welder, but for some reason you can't use a welder on it anymore.

**What did you expect to happen**: 
Try to dissasemble one.

**What happened instead**:
Nothing

**Why is this bad/What are the consequences**:
not being able to do something that you should be able to is bad

**Steps to reproduce the problem**:
Try to dissasemble a Delivery Chute and fail doing so.

**When did the problem start**:
27.02.2020

### PR DESCRIPTION
**Pull request by Citinited**
Title

Just some moar bugfixing

Fixes #13035
Fixes #13023
Fixes #12997
Fixes #11960

:cl:
fix: You can set fire to thermite with a welder again
fix: Setting light to thermite on walls will no longer leave a black box behind
fix: You can disassemble delivery chutes again
fix: Disassembling disposals bins no longer turns them into chutes
fix: You can open airlocks with fireaxes again
/:cl:

### CODE REVIEW

**Comment by Ty-Omaha** [comment]
your last CL entry is out of the CL 

> /🆑 Disassembling disposals bins no longer turns them into chutes

**Comment by Citinited** [comment]
Oops, not sure how I did that 😄

File: code/modules/recycling/sortingmachinery.dm [hunk 1]
```diff
@@ -345,21 +344,31 @@
...
+		return
+	if(!I.tool_use_check(user, 0))
+		return
+	WELDER_ATTEMPT_FLOOR_SLICE_MESSAGE
+	if(I.use_tool(src, user, 20, volume = I.tool_volume))
+		WELDER_FLOOR_SLICE_SUCCESS_MESSAGE
+		var/obj/structure/disposalconstruct/C = new (src.loc)
+		C.ptype = deconstructs_to
+		C.update()
+		C.anchored = 1
```
Comment (farie82) [inline code/modules/recycling/sortingmachinery.dm]: TRUE

File: code/modules/recycling/sortingmachinery.dm [hunk 2]
```diff
@@ -273,8 +273,7 @@
 	icon_state = "intake"
 	required_mode_to_deconstruct = 1
 	deconstructs_to = PIPE_DISPOSALS_CHUTE
-
-	var/c_mode = 0
+	var/c_mode = 0 //1 = we can deconstruct, 0 means we cannot
```
Comment (farie82) [inline code/modules/recycling/sortingmachinery.dm]: Maybe rename the var to make more sense? c_mode doesn't tell a reader much.

Comment (Citinited) [reply code/modules/recycling/sortingmachinery.dm]: Might as well even though I was just moving shit round 😄

**Pull request merged**